src/lib/encoder/test/DecoderTest.cpp

```diff
@@ -64,10 +65,12 @@ TEST_CASE( "DecoderTest/testDecode.4c", "[unit]" )
 {
 	// legacy format
 	MakeTempDirectory tempdir;
+	ConfigScope cs(4);
```

Comment:
...having a static (threadlocal in this case, but static) config means we need to lean on RAII to put things back in our tests.

As a bonus, the RAII helper can be how we set the config to begin with. Ergo: `ConfigScope`.